src/Core/Components/Splitter/FluentSplitter.razor.cs

```diff
@@ -57,6 +57,27 @@ public partial class FluentSplitter : FluentComponentBase
     [Parameter]
     public string? Panel2Size { get; set; }
 
+    /// <summary>
+    /// Gets or sets the minimum size for the left/top panel.
+    /// Needs to be a valid css size like '50%' or '250px'
+    /// </summary>
+    [Parameter]
+    public string? Panel1MinSize { get; set; }
+
+    /// <summary>
+    /// Gets or sets the minimum size for the right/bottom panel.
+    /// Needs to be a valid css size like '50%' or '250px'
+    /// </summary>  
+    [Parameter]
+    public string? Panel2MinSize { get; set; }
+
+    /// <summary>
+    /// Gets or sets the size of the splitter bar in pixels.
+    /// Default is 8
+    /// </summary>
+    [Parameter]
+    public int BarSize { get; set; } = 8;
```

Comment:
Because it needs to be a discrete value that can be used in the script code like this:

```
this.style.gridTemplateColumns = `${slot1fraction}fr ${median}px ${slot2fraction}fr`;
```
(or gridTemplateRows when direction is "column")